src/audio/kpb.c

```diff
@@ -808,7 +816,7 @@ static int kpb_prepare(struct comp_dev *dev)
 	}
 #endif /* CONFIG_IPC_MAJOR_4 */
 
-	if (!kpb->sel_sink || !kpb->host_sink) {
+	if (!kpb->sel_sink) {
 		comp_info(dev, "kpb_prepare(): could not find sinks: sel_sink %p host_sink %p",
 			  kpb->sel_sink, kpb->host_sink);
```

Comment:
the text should be changed too. And BTW it probably should be at least `comp_warn()` if not outright `comp_err()`